scripts/drone/create-release-tarball.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh -e
```

Comment:
We can remove this file as now release archive is generated by the task runner.

Reply:
Good point, I hope one of the colleagues will be able to take care of this since I am out of time and will be on holiday for the remainder of the sprint.